documents/2-RTB/Documentazione_Esterna/Glossario.typ

```diff
@@ -0,0 +1,425 @@
+
+#pagebreak()
+
+= #LTG("E")
+#pagebreak()
+
+= #LTG("F")
+#pagebreak()
+
+= #LTG("G")
+
+== #LTG("GitHub")
+
+GitHub è una piattaforma di sviluppo collaborativo basata su Git, un sistema di controllo delle versioni distribuito. È
+ampiamente utilizzato da sviluppatori di software e team di sviluppo per gestire progetti, tenere traccia delle
+modifiche al codice sorgente, coordinare il lavoro tra i membri del team e facilitare la collaborazione su larga scala
+
+== #LTG("Glossario")
+
+Elenco strutturato di termini tecnici o specializzati, ognuno corredato dalla propria definizione o spiegazione. Questo
+strumento aiuta a migliorare la comunicazione tra le varie parti coinvolte in un progetto, riducendo le ambiguità e
+garantendo una comprensione condivisa dei termini utilizzati in un determinato contesto
+
+== #LTG("GUI (Graphical User Interface)")
+
+Una GUI (acronimo di Graphical User Interface, in italiano Interfaccia Grafica Utente) è un'interfaccia che consente
+agli utenti di interagire con un sistema informatico o un software attraverso elementi visivi e grafici, anziché
+utilizzare solo comandi testuali o linee di comando.
+
+#pagebreak()
+
+= #LTG("H")
+#pagebreak()
+
+= #LTG("I")
+#pagebreak()
+
+= #LTG("J")
+
+== #LTG("Jira")
+
+Jira è uno strumento software sviluppato da Atlassian, utilizzato per la gestione di progetti, il monitoraggio dei
+problemi e il controllo dello sviluppo agile. È particolarmente popolare tra i team di sviluppo software, ma è anche
+ampiamente utilizzato in altri ambiti aziendali.
+
+#pagebreak()
+
+= #LTG("K")
+
+== #LTG("Key Performance Indicator (KPI)")
+
+I Key Performance Indicator (KPI) sono misure quantitative utilizzate per valutare le prestazioni di un'organizzazione,
+di un progetto o di un processo rispetto agli obiettivi prefissati. I KPI forniscono un modo oggettivo per monitorare e
+valutare il successo di un'attività, identificare le aree di miglioramento e prendere decisioni informate per
+ottimizzare le prestazioni
+
+#pagebreak()
+
+= #LTG("L")
+
+== #LTG("LaTeX")
+
+LaTeX è un linguaggio di markup e un sistema di preparazione di documenti utilizzato principalmente per la produzione di
+documenti scientifici, accademici e tecnici, ma anche per altri tipi di pubblicazioni come libri, tesi, articoli e
+presentazioni. È basato su TeX, un sistema di tipografia creato da Donald Knuth negli anni '70, ma LaTeX aggiunge una
+serie di comandi e pacchetti che ne semplificano l'utilizzo.
+
+#pagebreak()
+
+= #LTG("M")
+
+== #LTG("Milestone")
+
+Le Milestone sono strumenti utilizzati nella gestione dei progetti per segnare punti specifici lungo la timeline di un
+progetto. Questi punti possono segnalare ancore come una data di inizio e di fine del progetto, o la necessità di una
+revisione. In molti casi, le milestone, non incidono sulla durata del progetto. Si concentrano invece sui principali
```

Comment:
Rimuoverei il punto dopo "progetto". 
Ad esempio: "In molti casi, le milestone, non incidono sulla durata del progetto, ma si concentrano invece sui principali..."

Reply:
done